apps/fabric-website/scripts/createFlightConfig.js

```diff
@@ -0,0 +1,63 @@
+/*
+  Creates a configuration JS file that exposes the build ID and base URL that the site should load.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const { argv, logger } = require('@uifabric/build/just-task').just;
+
+module.exports = function createFlightConfigTask() {
+  return function createFlightConfigTask() {
+    let date = new Date();
+    let today = date.getFullYear() + ('0' + (date.getMonth() + 1)).slice(-2) + ('0' + date.getDate()).slice(-2);
+
+    let configsToGenerate = ['fabric-website-internal-prod-config', 'fabric-website-internal-df-config'];
+
+    // Produces date string of the form yyyMMdd, e.g. 20180701
```

Comment:
```suggestion
    // Produces date string of the form yyyyMMdd, e.g. 20180701
```

Reply:
And does this comment belong up a couple lines?

Reply:
oh yes, was moving things around!